LayoutTests/ipc/swift-receiver.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html><!-- webkit-test-runner [ IPCTestingAPIEnabled=true ] -->
+<title>Test that a Swift receiver round-trips a message</title>
+<script src="../resources/testharness.js"></script>
+<script src="../resources/testharnessreport.js"></script>
+<body>
+<script>
+promise_test(async (t) => {
```

Comment:
not really specific to this PR but I’m just curious why these types of tests are layout tests instead of API tests given they’re 100% logic based and not really visual